scripts/post-update.sh

```diff
@@ -268,6 +268,12 @@ if (( $PRE_0_0_16 > 0 )); then
 
     echo 192.168.2.2 > /home/pi/static-ip.conf
 
+    # remove any line containing 'enable_uart=' from /boot/config.txt
+    sudo sed -i '/enable_uart=/d' /boot/config.txt
+
+    # append 'enable_uart=1' line to /boot/config.txt
+    echo "enable_uart=1" | sudo tee -a /boot/config.txt
```

Comment:
why not `echo "enable_uart=1" >> /boot/config.txt`?

Reply:
The file is owned by root. The script is run as pi.

sudo echo > file, runs echo as sudo but the permissions are still the pi user.

Reply:
Ok, that makes sense